yeonjae/programmers-2/행렬의곱셈.js

```diff
@@ -0,0 +1,30 @@
+function solution(arr1, arr2) {
+  const answer = [];
+
+  for (let i = 0; i < arr1.length; i++) {
+    let tempArr = arr1[i];
+    answer.push([]);
+
+    for (let j = 0; j < arr2[0].length; j++) {
+      let sum = 0;
+
+      for (let z = 0; z < arr2.length; z++) {
+        sum += tempArr[z] * arr2[z][j];
+      }
+      answer[i].push(sum);
+    }
+  }
+
+  return answer;
+}
```

Comment:
```suggestion
function solution(arr1, arr2) {
  const answer = [];

  for (let i = 0; i < arr1.length; i++) {
    const tempArr = arr1[i];
    answer.push([]);

    for (let j = 0; j < arr2[0].length; j++) {
      let sum = 0;

      for (let z = 0; z < arr2.length; z++) {
        sum += tempArr[z] * arr2[z][j];
      }
      answer[i].push(sum);
    }
  }

  return answer;
}
```
- 풀이잘봤습니다~
- tempArr은 let이 아닌 const로 선언하는게 좋을 것 같습니다 :)

Reply:
앗 이부분을 놓쳤었네요ㅜㅠㅜㅜ 넵!